src/cpp/r/include/r/RCntxt.hpp

```diff
@@ -81,14 +81,18 @@ class RCntxt: public RCntxtInterface
    // NOTE: 'srcref()' attribute may be set to <in-bc-interp>
    // symbol for contexts associated with bytecode evaluation!
    bool isNull() const;
```

Comment:
Reordered to match the order defined in the R sources. Not strictly necessary, but makes cross-referencing a bit easier.